.github/workflows/tag-release-version.yml

```diff
@@ -12,6 +12,9 @@ jobs:
     if: github.event.pull_request.merged == true
     runs-on: ubuntu-latest
     steps:
+      - name: Ensure Changelog
+        run: gh pr view ${{github.event.pull_request.number}} --json files -q '.files[].path' | grep CHANGELOG.md
```

Comment:
Is this command available on ubuntu by default or we need to install? Maybe some authentication is required to access the PR or public repos does not require that? 🤔 

I was actually thinking to add a step in between that checks if we have `Unreleased` section in the changelog, but if this command works then it will be better.

Reply:
Afaik it's preinstalled on GitHub runners. PR access should also be possible.

The caveat is, that this will throw an error (exit code 1) when the changelog has not been modified, which is treated as a workflow failure.

Reply:
> The caveat is, that this will throw an error (exit code 1) when the changelog has not been modified, which is treated as a workflow failure.

That means the `failure()` check will not work to notify slack team?

We could try moving this step as a separate workflow, maybe that works out?

Probably the easier would be to just have a check for `Unreleased` section. We have a regex in `updateChangelog.js` that can be reused:
```ts
const changelogVersionRegExp = new RegExp(
  `\\[(development|develop|unreleased|${version})${optionalBetaOrRc}.*`,
  'gi',
);
```

Reply:
> That means the failure() check will not work to notify slack team?

It will work - as it is now, a missing changelog will result in a slack notification. I'm not sure if we want that (open for discussion) but IMO it's better to have half of a release.

I have not yet found a clean way to stop the execution without causing a failure, apart from splitting it into a separate job and adding a condition to the existing one.